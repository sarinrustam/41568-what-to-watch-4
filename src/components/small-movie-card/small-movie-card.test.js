import React from "react";
import rerender from "react-test-renderer";

import SmallMovieCard from "./small-movie-card.jsx";

const movie = {
  id: 3,
  title: `No country for old men`,
  img: `img/no-country-for-old-men.jpg`
};

const onMouseOverHandler = () => {};
const onMovieTitleClickHandler = () => {};

describe(`Render SMC`, () => {
  it(`SMC should render movie`, () => {
    const tree = rerender
    .create(<SmallMovieCard
      movie={movie}
      onMouseOver={onMouseOverHandler}
      onMovieClick={onMovieTitleClickHandler}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
