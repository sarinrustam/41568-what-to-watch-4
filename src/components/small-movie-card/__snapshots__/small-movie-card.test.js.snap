// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SMC SMC should render movie 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <video
      crossOrigin="anonymous"
      height="175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[Function]}
    >
      Pulp Fuction
    </a>
  </h3>
</article>
`;
